### ISSUE TITLE
[HANDS-ON BUG]: Unit 3: while pushing to the hub: KeyError: 'email'

### ISSUE THREAD

**Issue by ValentinGuigon** [huggingface/deep-rl-class#548]
Hi, 

I'm facing a _KeyError: 'email'_ error when running the following command, i.e., pushing to the hub on the Colab for Unit 3. I'm not so sure what triggered it, all the previous hands-on were running fine.

`!python -m rl_zoo3.push_to_hub  --algo dqn  --env SpaceInvadersNoFrameskip-v4  --repo-name dqn-SpaceInvadersNoFrameskip-v4  -orga ValentinGuigon  -f logs/`

Here is the output:
```
2024-07-08 02:57:35.221860: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:9261] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2024-07-08 02:57:35.221925: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:607] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2024-07-08 02:57:35.223584: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1515] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2024-07-08 02:57:35.232245: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2024-07-08 02:57:37.269536: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Loading latest experiment, id=2
Loading logs/dqn/SpaceInvadersNoFrameskip-v4_2/SpaceInvadersNoFrameskip-v4.zip
A.L.E: Arcade Learning Environment (version 0.8.1+53f58b7)
[Powered by Stella]
Stacking 4 frames
Wrapping the env in a VecTransposeImage.
Uploading to ValentinGuigon/dqn-SpaceInvadersNoFrameskip-v4, make sure to have the rights
ℹ This function will save, evaluate, generate a video of your agent,
create a model card and push everything to the hub. It might take up to some
minutes if video generation is activated. This is a work in progress: if you
encounter a bug, please open an issue.
/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_deprecation.py:131: FutureWarning: 'Repository' (from 'huggingface_hub.repository') is deprecated and will be removed from version '1.0'. Please prefer the http-based alternatives instead. Given its large adoption in legacy code, the complete removal is only planned on next major release.
For more details, please read https://huggingface.co/docs/huggingface_hub/concepts/git_vs_http.
  warnings.warn(warning_message, FutureWarning)
/content/hub/dqn-SpaceInvadersNoFrameskip-v4 is already a clone of https://huggingface.co/ValentinGuigon/dqn-SpaceInvadersNoFrameskip-v4. Make sure you pull the latest changes with `repo.git_pull()`.
WARNING:huggingface_hub.repository:/content/hub/dqn-SpaceInvadersNoFrameskip-v4 is already a clone of https://huggingface.co/ValentinGuigon/dqn-SpaceInvadersNoFrameskip-v4. Make sure you pull the latest changes with `repo.git_pull()`.
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.10/dist-packages/rl_zoo3/push_to_hub.py", line 410, in <module>
    package_to_hub(
  File "/usr/local/lib/python3.10/dist-packages/rl_zoo3/push_to_hub.py", line 201, in package_to_hub
    repo = Repository(repo_local_path, clone_from=repo_url)
  File "/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_validators.py", line 114, in _inner_fn
    return fn(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_deprecation.py", line 132, in inner_f
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/huggingface_hub/repository.py", line 545, in __init__
    git_email = user["email"]
KeyError: 'email'
```

In case you need the link to the Colab file, here it is:
https://colab.research.google.com/drive/1LyHDsV0KxcbqH4a6RkrIRh1TfubFP3zZ#scrollTo=Ygk2sEktTDEw

Thank you

**Comment by joaquinfeltes** [comment]
I'm having this same issue. I also saw [this issue](https://github.com/huggingface/deep-rl-class/issues/539.) that seems related.
I used all the provided solutions (with my hugging face account), I created the yml file and the logs folder was created also. Could this be a problem with a package version?
I saw [this](https://discuss.huggingface.co/t/why-am-i-getting-keyerror-email/89188) error that also seems related.
Thanks for the help!

Here is my notebook: https://colab.research.google.com/drive/1QCCXRfrapO-ntjJb-HrUoP4BFktvo6cS?usp=sharing
